python/ray/widgets/render.py

```diff
@@ -0,0 +1,39 @@
+import pathlib
+from typing import List
+
+from ray.util.annotations import DeveloperAPI
+
+
+@DeveloperAPI
+class Template:
+    """Class which provides basic HTML templating."""
+
+    def __init__(self, file: str):
+        with open(pathlib.Path(__file__).parent / "templates" / file, "r") as f:
+            self.template = f.read()
+
+    def render(self, **kwargs) -> str:
+        """Render a template by replacing instances of `{{ key }}` with `value`
+        from the keyword arguments.
+
+        Returns
```

Comment:
We use google style python docstrings btw, could we edit the style of the docstrings to match?

Reply:
:facepalm: sorry about this! I've reformatted the docstrings accordingly!